### PR TITLE
auditor dashboard added

### PR DESCRIPTION
**Pull request by Ankit017-sri**

### CODE REVIEW

File: frontend/src/auditor/AuditorDashboard.js [hunk 1]
```diff
@@ -18,21 +19,21 @@ const AuditorDashboard = () => {
       loanType: "bullet",
     },
   ]);
-  //const path = useHistory();
+  const path = useHistory();
 
-  //   useEffect(() => {
```
Comment (Dygnify) [inline frontend/src/auditor/AuditorDashboard.js:23]: Remove unwanted code instead of commenting it

File: frontend/src/auditor/OpportunityDetailsAuditor.js [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+  const hash = target?.opportunity_info;
+
+  const info = ExtractIPFSdataFromHash(hash);
+
+  return (
+    <>
+      <style>{"body { background-color: #7165e3 }"}</style>
+      <Box
+        sx={{
+          height: "90px",
```
Comment (Dygnify) [inline frontend/src/auditor/OpportunityDetailsAuditor.js:43]: for this time as this is not our final ui duplication is okay but for future integration do it using reusable components

File: frontend/src/borrower/Borrower.js [hunk 3]
```diff
@@ -1,7 +1,9 @@
-import { React, useState, useEffect, useHistory } from "react";
+import { React, useState, useEffect } from "react";
 import { Box, Button, Typography, Stack, Divider, Card } from "@mui/material";
 import OpportunityTable from "./OpportunityTable.js";
 import DrawdownCard from "./DrawdownCard.js";
+import { useHistory } from "react-router-dom";
```
Comment (Dygnify) [inline frontend/src/borrower/Borrower.js:5]: Why this change? i don't see any further use of 'useHistory'

File: frontend/src/components/transaction/TransactionHelper.js [hunk 4]
```diff
@@ -2,12 +2,12 @@ import { ethers } from "ethers";
 import dygnifyStaking from "../../artifacts/contracts/DygnifyStaking.sol/DygnifyStaking.json";
 import dygnifyToken from "../../artifacts/contracts/DygnifyToken.sol/DygnifyToken.json";
 import { requestAccount } from "../navbar/NavBarHelper";
-import opportunityOrigination from "../../artifacts/contracts/opportunityOrigination.sol/opportunityOrigination.json";
+import opportunityOrigination from "../../artifacts/contracts/protocol/OpportunityOrigination.sol/OpportunityOrigination.json";
 
 const dygnifyStakingAddress = "0xCF1709F792c209Bf8fF1294aD9deaF0dfE44e9F6";
 const token = "0x9C80225f50E1be2fa8b1f612616d03Bc9a491107";
 const opportunityOriginationAddress =
-  "0x474FE9bCBe747a22ACee6e9A2E18d4EBaa552d94";
```
Comment (Dygnify) [inline frontend/src/components/transaction/TransactionHelper.js:10]: Move this to env file and use from there

File: frontend/src/components/transaction/TransactionHelper.js [hunk 5]
```diff
@@ -233,3 +233,93 @@ export async function getOpportunitysOf() {
...
+    }
+  } catch (error) {
+    console.log(error);
+  }
+
+  return 0;
+}
+
+export async function voteOpportunity(id, vote) {
+  if (typeof window.ethereum !== "undefined") {
```
Comment (Dygnify) [inline frontend/src/components/transaction/TransactionHelper.js:279]: use exception handling in this function

**Review by Dygnify — CHANGES_REQUESTED** [review]

File: frontend/src/auditor/AuditorDashboard.js (hunk 1 above)
Comment (Ankit017-sri) [reply frontend/src/auditor/AuditorDashboard.js:23]: no thats not unwanted code may be it has previous version
i am using that piece of code for oppurtunity

File: frontend/src/auditor/OpportunityDetailsAuditor.js (hunk 2 above)
Comment (Ankit017-sri) [reply frontend/src/auditor/OpportunityDetailsAuditor.js:43]: ok

File: frontend/src/borrower/Borrower.js (hunk 3 above)
Comment (Ankit017-sri) [reply frontend/src/borrower/Borrower.js:5]: Its been used to redirect to oppurtunity details 
but that code is commented down

File: frontend/src/components/transaction/TransactionHelper.js (hunk 5 above)
Comment (Ankit017-sri) [reply frontend/src/components/transaction/TransactionHelper.js:279]: this function is given by Anirudha
i'll add try catch block

File: frontend/src/components/transaction/TransactionHelper.js [hunk 6]
```diff
@@ -4,6 +4,10 @@ import dygnifyToken from "../../artifacts/contracts/protocol/DygnifyToken.sol/Dy
 import { requestAccount } from "../navbar/NavBarHelper";
 import opportunityOrigination from "../../artifacts/contracts/protocol/OpportunityOrigination.sol/OpportunityOrigination.json";
 
+const dygnifyStakingAddress = "0xCF1709F792c209Bf8fF1294aD9deaF0dfE44e9F6";
```
Comment (Dygnify) [inline frontend/src/components/transaction/TransactionHelper.js:7]: make same changes for dygnifyStakingAddress and token

**Review by Dygnify — CHANGES_REQUESTED** [review]
